src/main/io/serial_msp.c

```diff
@@ -587,6 +590,47 @@ static uint32_t packFlightModeFlags(void)
     return junk;
 }
 
+
+typedef struct pgToMSPMapEntry_s {
+    pgn_t pgn;
+    uint8_t mspId;
+    uint8_t mspIdForSet;
+} pgToMSPMapEntry_t;
+
```

Comment:
BTW: same section trick used in PG_REGISTER may be used for pgn->ms mapping.
It may help with code isolation (and it will help a lot if some config is conditionally compiled)